src/org/traccar/reports/ReportUtils.java

```diff
@@ -61,19 +61,19 @@ public static void checkPeriodLimit(Date from, Date to) {
         }
     }
 
-    public static String getDistanceUnit(long userId) {
+    public static String getDistanceUnit(Long userId) {
```

Comment:
Why are we changing everything to `Long` class?

Reply:
Rollbacked, done.